src/common/components/WalletBalanceInput/CostDisplay.tsx

```diff
@@ -0,0 +1,16 @@
+import { HStack } from '@chakra-ui/react';
+
+import { Text } from '../Typography';
+
+export const CostDisplay = ({ totalCost, balanceAmount, symbol }) => {
```

Comment:
Could we describe the type of each prop? Better to avoid `any` type in the future